modules/luci-mod-admin-full/luasrc/view/admin_system/packages.htm

```diff
@@ -20,7 +20,7 @@
 local space_free  = fstat and fstat.bfree  or 0
 local space_used  = space_total - space_free
 
-local used_perc = math.floor(0.5 + ((space_total > 0) and ((100 / space_total) * space_used) or 100))
+local used_perc = math.floor(0,5 + ((space_total > 0) and ((100 / space_total) * space_used) or 100))
```

Comment:
0,5 here looks wrong. That is not a decimal number, but 0 and 5
Should be 0.5, so this change is wrong

see https://github.com/openwrt/luci/pull/1768#issuecomment-384653945

Reply:
You are right.